src/Repository/RawPaymentMethodRepository.php

```diff
@@ -128,6 +128,7 @@ private function getPaymentMethodsData()
             ['name' => 'giropay', 'label' => 'GiroPay', 'icon' => 'Giropay.svg', 'template' => '', 'is_payment_method' => '1'],
             ['name' => 'kbcpaymentbutton', 'label' => 'KBC', 'icon' => 'KBC.svg', 'template' => '', 'is_payment_method' => '1'],
             ['name' => 'bancontactmrcash', 'label' => 'Bancontact', 'icon' => 'Bancontact.svg', 'template' => '', 'is_payment_method' => '1'],
+            ['name' => 'blik', 'label' => 'Blik', 'icon' => 'Blik.svg', 'template' => '', 'is_payment_method' => '1'],
```

Comment:
you also need to create a upgrade file to insert blik payment method when merchant has the plugin already installed